tools/code_format/check_format.py

```diff
@@ -895,6 +895,15 @@ def clangFormat(file_path):
     return ["clang-format rewrite error: %s" % (file_path)]
   return []
 
+# Fix for https://github.com/envoyproxy/envoy/issues/10535
+def checkProtoiValidation(file_path):
+  command = "grep -qrnwl -e 'min_bytes' %s" % (file_path)
```

Comment:
You don't need to literally grep, see other examples in this file of how `regex` is used instead to avoid having to shell out.

Reply:
done with "re" module as per inputs.